### PR TITLE
account settings page

### PR DESCRIPTION
**Pull request by inflrscns**
https://github.com/django-wiki/django-wiki/issues/490
Adds a simple page where users can change their password and email address. It looks like this:
screenshot

I also fixed a small typo ("ArticlManagerTests" -> "ArticleManagerTests")

### CODE REVIEW

File: wiki/templates/wiki/base_site.html [hunk 1]
```diff
@@ -65,6 +65,12 @@
                       {% trans "Log out" %}
                     </a>
                   </li>
+                  <li>
+                    <a href="{% url 'wiki:update' %}">
```
Comment (benjaoming) [inline wiki/templates/wiki/base_site.html]: The url name "update" is a little too generic :)

File: wiki/urls.py [hunk 2]
```diff
@@ -42,6 +42,7 @@ class WikiURLPatterns(object):
     signup_view_class = accounts.Signup
     login_view_class = accounts.Login
     logout_view_class = accounts.Logout
+    settings_view_class = accounts.Update
```
Comment (benjaoming) [inline wiki/urls.py]: "settings_view_class" ? Could also be "profile_update_view_class"

**Comment by benjaoming** [comment]
Thanks! We could definitely use a bit more "batteries included" by having this option for users to change their passwords.

I think that the link shouldn't be shown when account handling is off. So the template should have a condition such that it can check the value of settings.ACCOUNT_HANDLING

A couple of comments above, too, and I did a PR for your code regarding making a proper barrier when account handling is off.

**Comment by benjaoming** [comment]
Could you write a small test, too?

**Comment by inflrscns** [comment]
Okay, I think that should cover everything. Let me know if it needs any more changes!

**Comment by benjaoming** [comment]
Awesome work, I somehow got mislead to think there was something missing in this PR. I had notifications from Github off for a long time due to offline-like conditions.

So, going to merge this. Might fix a couple of pep8-like things, but this looks great nonetheless! Thanks! Will come out in next release!

**Pull request merged**